docs/sources/installation/ubuntulinux.md

```diff
@@ -1,13 +1,13 @@
-page_title: Installation on Ubuntu 
-page_description: Instructions for installing Docker on Ubuntu. 
```

Comment:
Have you checked if the meta headers are correctly used after removing the spaces? (not behind my PC right now)